### PR TITLE
md-raid: add detection for check-state

### PR DESCRIPTION
**Pull request by vidschofelix**
atm checks turn md-service into warning state, bc. they come with "finish" and "speed" values. Checks are "By default, run at 00:57 on every Sunday, but do nothing unless the day of the month is less than or equal to 7 [...]". 
A check is no resync or recovery, it's just a periodic check and shouldn't degrade the state of the raid.

### CODE REVIEW

**Comment by jherbel** [comment]
Hi Felix,
thanks for your pull request. Before we can look at it more closely, all the tests need to work. See https://github.com/tribe29/checkmk/blob/master/CONTRIBUTING.md#how-to-execute-tests for information on how to run them locally.
Best
Jörg

**Comment by vidschofelix** [comment]
Hi Jörg,
thanks. 
Codestyle is fixed.

**Comment by jherbel** [comment]
Hi Felix,
thank you, we will take a look at your PR and check if and how we can implement into the official code mase.
Internal reference: CMK-5058
Best
Jörg

File: checks/md [hunk 1]
```diff
@@ -178,24 +193,29 @@ def check_md(item, _no_params, parsed):
...
+        
     if "finish" in data:
         infotexts.append("Finish: %s" % data["finish"])
 
     if "speed" in data:
         infotexts.append("Speed: %s" % data["speed"])
-
-    if infotexts:
+        
+    if "check_state" in data:
```
[the remark this answers is not in the document]
Comment (vidschofelix) [reply checks/md]: absolutely. fixed

**Comment by LiraLemur** [comment]
Hello @vidschofelix ! 
I am Sara, Checkmk's Community Manager, and we need your help regarding this contribution. 

Could you please contact me at community@checkmk.com regarding this? 
Looking forward to your reply and thank you for your prompt response :)

**Pull request closed without merging**